Makefile

```diff
@@ -121,7 +121,7 @@ install-wrk:
 	cd ./vendor/wrk ; (ls ./wrk 2>/dev/null || make >install_wrk.log)
 
 .PHONY: test
-test: generate lint
+test: check-generate lint
```

Comment:
we want to run `generate` here. `check-generate` takes far too long

Reply:
@zhenghuiwang suggested since i had issue with easy json while tests passed locally but failed on travis

Reply:
yeah you sometimes have to run `check-generate` locally but it can take upto 2 minutes instead of 10 seconds. 

This is just something we have to do sometimes to unbreak the code :( 

Something about easyjson is order dependent and you have to re-generate it sometimes.